libs/entitlement/src/main/java/module-info.java

```diff
@@ -19,10 +19,10 @@
 
     exports org.elasticsearch.entitlement.runtime.api;
     exports org.elasticsearch.entitlement.runtime.policy;
-    exports org.elasticsearch.entitlement.runtime.policy.entitlements to org.elasticsearch.server;
     exports org.elasticsearch.entitlement.instrumentation;
     exports org.elasticsearch.entitlement.bootstrap to org.elasticsearch.server;
     exports org.elasticsearch.entitlement.initialization to java.base;
+    exports org.elasticsearch.entitlement.runtime.policy.entitlements to org.elasticsearch.server;
```

Comment:
Why is this moved? Seems it was better next to the related package (though I don't see why the policy package is needed outside of entitlements).

Reply:
<Sigh> IntelliJ auto-formatting presumably.